src/component.cpp

```diff
@@ -41,6 +41,7 @@ namespace libcellml {
  */
 struct Component::ComponentImpl
 {
+    Component *mQ = nullptr;
```

Comment:
is `mQ` a standard name to designate the instance of the API class? `mQ` certainly doesn't mean anything to me. If anything, I might use something like `mOwner`?

Reply:
`mQ` is the return of `mPimpl` I wouldn't say it was a standard though.

Reply:
Following QT here, We have used `mValidator` when in the pimpl for the Validator class for instance.

Reply:
I don't think `mOwner` conveys the correct meaning to what it is.

Reply:
Not sure why `mOwner` is not fine since it's definitely the owner of an instance of `ComponentImpl`. Anyway, I would certainly be happy with being consistent with what has already been done for the `Validator` class, i.e. use `mComponent` instead of `mQ`.

Reply:
40ce4fa